packages/coze-js/src/ws-tools/wavtools/lib/wav_stream_player.d.ts

```
@@ -0,0 +1,73 @@
+     * @param {{sampleRate?: number}} options
+     * @returns {WavStreamPlayer}
+     */
+    constructor({ sampleRate }?: {
+        sampleRate?: number;
+    });
+    scriptSrc: any;
+    sampleRate: number;
+    context: any;
+    stream: any;
+    analyser: any;
+    trackSampleOffsets: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)